astropy/wcs/utils.py

```diff
@@ -71,6 +73,37 @@ def solar_system_body_representation_type(
     )
 
 
+def north_pole_angle(pixel, wcs, ddec=0.1 * u.arcsec):
+    """
+    Computes counterclockwise angle between positive x-axis and sky North
+
+    Parameters
+    ----------
+    pixel : tuple, required.
+        reference pixel in image from which to find angle
+    wcs : `~astropy.wcs.WCS`
+    ddec : `~astropy.coordinates.Angle`, optional, default 0.1 arcseconds.
+        small angular offset for computing direction of north
+        no need to change unless very close to north pole
```

Comment:
Your sentence was incomplete, so I tried to fill it in but was I correct?

```suggestion
    ddec : `~astropy.coordinates.Angle`, optional
        Small angular offset for computing direction of North.
        No need to change this value unless pixel is very close to North pole.
        Default 0.1 is arcseconds.
```